docs/testing-teal.md

```diff
@@ -13,29 +13,58 @@
 - [Parser](../packages/runtime/src/parser): Reads TEAL code and converts it to a list of opcodes which are executable by the interpreter. If any opcode/data in teal code is invalid, parser will throw an error.
 - [Interpreter](../packages/runtime/src/interpreter): Executes the list of opcodes returned by parser and updates stack after each execution. At the end of execution, if the stack contains a single non-zero uint64 element then the teal code is approved, and transaction can be executed.
 
+## Block Rounds/Height
+In runtime User can decide the round for each transaction or group of transactions. By default round is set to `2`.
+Block rounds for runtime can be set by using `runtime.setRound(round)`. We can retrieve the round by using `runtime.getRound()` function.<br />
+Example:
+
+      runtime.setRound(5);
+This means that current round is set to 5 and transaction will pass only if first valid round is less than 5 and last valid round is greater than 5 for that particular transaction.
+Note: Block round remains same until user changes it by using `runtime.setRound(round)`.
+
+## Flow of Testing
+In this section we will describe the flow of testing:
+- Prepare Accounts: First of all we will create accounts using `StoreAccount`.
+
+      const john = new StoreAccount(initialAlgo);
+      const bob = new StoreAccount(initialAlgo);
+- Prepare Runtime: After creating accounts we will create a runtime object with those accounts.
+
+      const runtime = new Runtime([john, bob]);
+- Set Rounds: Now we will set the rounds according to our requirement.
+
+      runtime.setRound(20);
+    This means that current round is set to 20 and transactions will pass only if their round range includes 20.
+- Create Apps/Assets: At this point we have set up everything, now we can create apps or assets or both and test our smart contracts. If we want to create an app, we can use `runtime.addApp()` funtion, similarly for asset we can use `runtime.addAsset()` function.
```

Comment:
```suggestion
- Create Apps/Assets: At this point we have set up everything, now we can create apps and/or assets and begin testing our smart contracts (present in `/fixtures/project/assets`). For creating a stateful application, we can use `runtime.addApp()` funtion, similarly for creating a new asset we can use `runtime.addAsset()` function.
```

Reply:
what do you mean my runtime is read ? @ratik21

Reply:
ah, you mean `ready`